srtcore/buffer.h

```diff
@@ -419,7 +426,7 @@ class CRcvBuffer
 
 private:
    CUnit** m_pUnit;                     // pointer to the protocol buffer
-   int m_iSize;                         // size of the protocol buffer
+   int m_iSize;                         // size of the protocol buffer in units
```

Comment:
This change is unnecessary. Size never means size in bytes by default. This thing is a container, and `m_iSize` is its size. If you really want meaningful comments here, state explicitly that you have an array of `CUnit*` items, and then a size of this array.

Reply:
Done. Also changed `m_iSize` to const.